sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -213,7 +213,8 @@ Duration getDefaultPollingInterval() {
         KeyRequestParameters parameters = new KeyRequestParameters()
             .setKty(createKeyOptions.getKeyType())
             .setKeyOps(createKeyOptions.getKeyOperations())
-            .setKeyAttributes(new KeyRequestAttributes(createKeyOptions));
+            .setKeyAttributes(new KeyRequestAttributes(createKeyOptions))
+            .setKeyReleasePolicy(createKeyOptions.getKeyReleasePolicy());
```

Comment:
Given the swagger is "release_policy", I think these should actually be ReleasePolicy instead of KeyReleasePolicy. That was in the original bug as well.

Reply:
No problem, thanks for pointing it out.